src/main/java/org/opentripplanner/routing/core/TraverseMode.java

```diff
@@ -80,7 +20,7 @@ public boolean isDriving() {
   }
 
   public boolean isCycling() {
-    return this == BICYCLE;
+    return this == BICYCLE || this == SCOOTER;
```

Comment:
Is this correct? Maybe we should change name of the method?

Reply:
Yes, this is correct